### ISSUE TITLE
This should be renamed to Jellypotato to be in line with previous literature

### ISSUE THREAD

**Issue by Jarry-TDWTF** [benzweig/jankyscroll#11]
Hi! at [WTDWTF](https://what.thedailywtf.com/) we have coined a term for this kind of scroll behaviour:

> Jellypotato
> _noun_
> Random page scrolling, typically caused by clicking an innocuous control such as a user avatar.

https://what.thedailywtf.com/post/565011

i think this nice plugin should be called `jellypotato.js`
please?

**Comment by ikeadrift** [comment]
I think you're talking about a more specific behavior. jankyscroll is a sad funhouse mirror reflection of non-native scrolling in general, not limited to interactions. you cannot Control the Scroll

**Comment by BenLubar** [comment]
There seems to be some confusion here. jankyscroll.js is _not_ an example of Jellypotato.

jankyscroll.js is an example of what happens when websites re-implement the browser functionality of scrolling. This can be seen in a lot of websites that didn't know about `overflow: auto` or maybe they wanted to have their own "web 2.0" scrollbar styles, and so they reimplemented it in JavaScript, usually ending up nearly as bad as jankyscroll.

Jellypotato is when the page reflows due to something that shouldn't have caused a page reflow, making whatever you were reading move. For example, on a forum we used, a horizontal rule in a user's bio would cause clicking their avatar to scroll to the very top of the page, causing infiniscroll to load the previous page, which showed the top of the previous page briefly before scrolling back down to the wrong location.

The major difference is what triggers the incorrect scrolling: In jankyscroll's case, the scrolling is triggered by scrolling. In Jellypotato's case, the scrolling is triggered by some action that should not cause the scroll position to change in the first place, although I would classify an image being added to the DOM and causing a reflow that pushes text off the screen as Jellypotato even if it was caused by a scroll event handler.
